repo/packages/H/history/0/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "history",
+  "version": "0.0.4",
+  "website": "https://github.com/mesosphere/dcos-history-service",
+  "maintainer": "support@mesosphere.io",
+  "description": "DCOS history application package",
```

Comment:
Could you expand on this a bit?  Why do users want this?  The description is searchable too.  Also, no need to mention it's a package here :)